clients/cpp-console/include/DialogManager.h

```diff
@@ -56,6 +57,8 @@ class DialogManager
     void StopKws();
     // Start a listening session that read audio stream from a wav file.
     void ListenFromFile();
+    // Get mute state of the default microphone.
+    bool IsMuted() { return (_muter) ? _muter->IsMuted() : false; };
```

Comment:
? looks wrong
should be
{ return (_muter ? _muter->IsMuted() : false) };
I think you misplaced a closing parenthesis

Reply:
I like to use parenthesis this way like they are used after "if". It works here, but we can change.